ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -645,15 +646,16 @@ public CompletableFuture<Void> setDisabled() {
       isPoolEnabled.set(false);
       subscribeConnectId.ifPresent(ethContext.getEthPeers()::unsubscribeConnect);
       pendingTransactionsListenersProxy.unsubscribe();
-      final PendingTransactions pendingTransactionsToSave = pendingTransactions;
+      final var saveOperation =
```

Comment:
Would prefer to see a concrete type here rather than `var`. It's not entirely clear what type will be used.